### PR TITLE
Adjustable UI for iPad

### PR DESCRIPTION
**Pull request by descorp**
### Changes

* make max DropIn width for iPad 0.85% or 375px
* add `Dimentions` object

Screenshot 2021-07-23 at 14 25 13

### CODE REVIEW

File: AdyenDropIn/Presentation/WrapperViewController.swift [hunk 1]
```diff
@@ -63,14 +74,21 @@ extension ModalViewController {
...
     }
 
     /// Enables any `UIViewController` to recalculate it's conten's size form modal presentation ,
     /// e.g `viewController.adyen.finalPresentationFrame(in:keyboardRect:)`.
     /// :nodoc:
     func finalPresentationFrame(in containerView: UIView, keyboardRect: CGRect = .zero) -> CGRect {
         var frame = containerView.bounds
+
+        if UIDevice.current.userInterfaceIdiom == .pad {
+            let width = min(frame.width * 0.85, 375 * UIScreen.main.scale)
```
Comment (mohammedDehairy) [inline AdyenDropIn/Presentation/WrapperViewController.swift:87]: (85 % of width or 375) is repeated in two places, can we put this in one place, like a global function or something?

Comment (descorp) [reply AdyenDropIn/Presentation/WrapperViewController.swift]: Done in `Dimentions` enum

File: Adyen/Helpers/UIViewHelpers.swift [hunk 2]
```diff
@@ -64,4 +64,12 @@ extension AdyenScope where Base: UIView {
         base.alpha = hidden ? 0 : 1
         base.adyen.updatePreferredContentSize()
     }
+
+    public var getMinimalSize: CGSize {
```
Comment (erenbesel) [inline Adyen/Helpers/UIViewHelpers.swift:68]: Naming it without get would be more Swifty`minimalSize`

Comment (descorp) [reply Adyen/Helpers/UIViewHelpers.swift]: 👍

**Review by erenbesel — APPROVED** [review]

**Review by mohammedDehairy — APPROVED** [review]

**Pull request merged**